components/list/list-item-drag-handle.js

```diff
@@ -274,6 +275,10 @@ class ListItemDragHandle extends LocalizeCoreElement(FocusMixin(RtlMixin(LitElem
 
 	}
 
+	_onMoveButtonMouseDown(e) {
+		e.preventDefault();
```

Comment:
This is needed to stop Safari from moving focus away from the button when it is clicked due to the frustrating position that Apple has taken.